src/annotator/sidebar.js

```diff
@@ -80,7 +80,11 @@ export default class Sidebar extends Guest {
       } else {
         // BucketBar is a "plugin" for legacy reasons and is now constructed here so
         // that the parent element can be passed into the constructor.
-        this.plugins.BucketBar = new BucketBar(frame, {}, this);
+        this.plugins.BucketBar = new BucketBar(
+          frame,
+          this.options.BucketBar,
```

Comment:
This could be changed to `config.BucketBar` to make it a little more obvious about where the value comes from. It will also be a step towards potentially removing `this.options` in future, which isn't used in that many remaining places.

Reply:
Thanks for the advice. Implemented.